tools/mutect/filter_mutect.nf

```diff
@@ -1,22 +1,20 @@
 #!/usr/bin/env nextflow
-nextflow.enable.dsl = 2
 
 // filter Mutect2 calls with GATK
 process FilterMutectCalls {
     // Set maximum memory
     memory '40 GB'
 
-    publishDir "${params.outdir}/filtered", mode: 'copy'
+    publishDir "${params.outdir}/vcf_filtered", mode: 'copy'
 
     input:
-    path unfiltered_vcf
     path mutect_idx
+    path unfiltered_vcf
+    path read_orientation_model
```

Comment:
Can we change read_orientation_model to artifact_likelihoods? I can make changes but also don't want to introduce inconsistencies with the submodule and params file naming convention.